### PR TITLE
CI: Azure Devops add sonarcloud analysis

### LINKED ISSUE #326: Setting up Jhipster-Core for Azure Pipelines
##### **Overview of the feature request**

Since Travis future is now uncertain, we should progressively migrate our CI to Azure Pipelines like we did for the generator or prettier-java.

##### **Motivation for or Use Case**

Travis is not doing well.
Azure Pipelines can have up to 10 parallel jobs (5 for Travis)
The sonar is currently broken [0% coverage](https://sonarcloud.io/dashboard?id=jhipster-core), this is due to the sonar addon on travis. (It can be easily fixed)

##### **Related issues or PR**

I worked on the ci migration and made it worked [here](https://github.com/Shaolans/jhipster-core/pull/9) but the pipeline needs to be configured beforehand:
- Install [sonarcloud extension](https://marketplace.visualstudio.com/items?itemName=SonarSource.sonarcloud)
- add a service connection for SonarCloud

- [ x] Checking this box is mandatory (this is just to show you read everything)

### PR DESCRIPTION
**Pull request by Shaolans**
Please make sure the below checklist is followed for Pull Requests.
  - [ ] [Travis tests](https://travis-ci.org/jhipster/jhipster-umk/pull_requests) are green
  - [ ] Tests are added where necessary
  - [ ] Documentation is added/updated where necessary
  - [ ] Coding Rules & Commit Guidelines as per our [CONTRIBUTING.md document](https://github.com/jhipster/jhipster-core/blob/master/CONTRIBUTING.md) are followed

Fix #326 
We just need to configure SonarCloud connection (this is why Azure is failing) on Azure and it should be good.

### CODE REVIEW

**Comment by MathieuAA** [comment]
Where are we on that? that'd be great to have it :)

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -40,7 +40,7 @@
 
 <!--
     You have to add it, we can't help you otherwise.
-    If you can't pass it, send it via mail at mathieu.aa@free.fr
```
Comment (MathieuAA) [inline .github/ISSUE_TEMPLATE.md]: Can you use the jhipster tech address instead? that'd be better :)

**Comment by Shaolans** [comment]
I believe that we just need to connect Sonar with Azure Pipelines plugin, however last time @pascalgrimaud did not have the credentials on sonarcloud.

**Comment by MathieuAA** [comment]
Maybe @jdubois has them?

File: .github/ISSUE_TEMPLATE.md (hunk 1 above)
Comment (Shaolans) [reply .github/ISSUE_TEMPLATE.md]: mathieu.aa@jhipster.tech ?

Comment (MathieuAA) [reply .github/ISSUE_TEMPLATE.md]: I believe it is `mathieu.abou-aichi@jhipster.tech`

**Comment by Shaolans** [comment]
I think @jdubois had to pass the rights to @pascalgrimaud but idk if this has been done

**Comment by MathieuAA** [comment]
The ReadMe file has already been updated

**Pull request closed without merging**